src/compile/utils/tail.ts

```diff
@@ -0,0 +1,7 @@
+export function new_tail(): string {
+	return '%%tail_head%%';
+}
+
+export function attach_head(head: string, tail: string): string {
+	return tail.replace('%%tail_head%%', head);
+}
```

Comment:
Since the new `dom` function `object_without_properties` needs to be invoked with the entire value as argument, it doesn't work to just append the tail anymore.

I'm not sure this is an optimal solution. Maybe there is another utility that could fit here?